README.md

```diff
@@ -198,16 +198,16 @@ let params = ChallengeParameters(threeDSServerTransactionID: response.transactio
                                  acsTransactionID: response.acs.transactionId,
                                  acsRefNumber: response.acs.referenceNumber,
                                  acsSignedContent: response.acs.signedContent)
-transaction?.doChallenge(challengeParameters: params, completion: { [weak self]  result in
+transaction?.doChallenge(challengeParameters: params, completion: { result in
      switch result {
       case .success(let authenticationResult):
          // handle authentication result. Checkout Payment Authorisation section.
       case .failure(let error):
          // handle failure scenario
        }
       // call close and cleanUp methods after challenge flow is completed. 
-      self?.transaction?.close()
-      self?.standalone3DSService?.cleanUp()
+      self.transaction?.close()
+      self.standalone3DSService?.cleanUp()
```

Comment:
I wouldn't prefer presenting the way of our SDK's integration in a way that might cause problems for merchants. Why don't we keep the first version?